src/test/unit/shorten-url.test.js

```diff
@@ -78,7 +78,7 @@ describe('shortenUrl', () => {
   });
 });
 
-describe('shortenUrl', () => {
+describe('expandUrl', () => {
```

Comment:
I was in here editing originally, and this was wrong. I reverted my other changes but not this one.